library/system/src/modules/FileChanges.rb

```diff
@@ -213,16 +213,24 @@ def CheckFiles(files)
       ret == :ok
     end
 
+    # Files that are really new
+    #
+    # @param files [Array<String>] candidate files that may be new
+    # @return [Array<String>]
+    def new_created_files(files)
```

Comment:
hmm, it is different naming convention and also in combination of CheckNew sounds for me a bit different. I actually notice it first time in second pull request, where it hit my eyes, but I let it up to you